engine-tests/src/tests/access_lists.rs

```diff
@@ -26,11 +26,13 @@ fn test_access_list_tx_encoding_decoding() {
         data: vec![0],
         access_list: vec![
             AccessTuple {
-                address: test_utils::address_from_hex("0x095e7baea6a6c7c4c2dfeb977efac326af552d87"),
+                address: test_utils::address_from_hex("0x095e7baea6a6c7c4c2dfeb977efac326af552d87")
```

Comment:
Should the `address` field of `AccessTuple` be the `Address` type instead of `H160`? Or does it make things awkward with passing this into SputnikVM if we do that?

Reply:
Yes. Because it is related to the Traits of SputnikVM.